tests/firewall/firewall-06-tls-sni-enforce/test.yaml

```diff
@@ -6,11 +6,24 @@ args:
 
 checks:
 - filter:
+    min-version: 8
+    count: 24
```

Comment:
this I think makes sense, the sig is
```
drop ip any any -> any any (msg:"Drop all else"; flow:stateless; sid:3;)
```
If there are no "real" packets, why raise alerts?